site/guide/model-documentation/work-with-content-blocks.qmd

```diff
@@ -3,27 +3,57 @@ title: "Work with content blocks"
 date: last-modified
 ---
 
-Make edits to your model documentation or validation reports by adding or removing content blocks directly in the online editor. 
+Make edits to your model documentation, validation reports, or ongoing monitoring plans by adding or removing content blocks directly in the online editor. 
 
 ## What are content blocks? 
 
-Content blocks provide you with sections that are part of a template. You can think of these sections as an empty canvas that you fill in with text and test results. Multiple sections are joined to create a longer document with a table of contents that has different heading and subheading levels, such as "1.," "1.1.," and so on. 
+Content blocks provide you with sections that are part of a template. 
 
-Types of content blocks: 
+- You can think of these sections as an empty canvas that you fill in with text and test results. 
+- Multiple sections are joined to create a longer document with a table of contents that has different heading and subheading levels, such as "1.," "1.1.," and so on. 
 
-Simple text block
-: Can be added anywhere on model documentation or validation reports and edited to include additional documentation in text format.
+#### Content block types
```

Comment:
Paraphrasing the wise person who cleaned up our glossary, these need to be added to our central glossary, not be hard-coded into different topics.

Reply:
That seems like an additional Story, and out of scope for this one. Can you make a related Story with your preferred acceptance criteria please? 🙏🏻

Reply:
NVM, quick glossary entry and single-sourced the block files for now:

<img width="1152" alt="Screenshot 2024-09-19 at 3 57 48 PM" src="https://github.com/user-attachments/assets/73f0c0e3-0ecf-4ae0-8175-4adc57f6ffa5">